senza/components/auto_scaling_group.py

```diff
@@ -70,11 +93,13 @@ def component_auto_scaling_group(definition, configuration, args, info, force, a
                 'Roles': instance_profile_roles
             }
         }
-        definition["Resources"][config_name]["Properties"]["IamInstanceProfile"] = {'Ref': logical_id}
+        definition["Resources"][config_name]["Properties"][
```

Comment:
I think it was the result of autopep8 in my editor. I will have a look at
it tomorrow.